bin/collator/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "astar-collator"
-version = "5.24.0"
```

Comment:
We have a new way of working - unless we plan to immediately do the release, crate versions & runtime spec versions, are only bumped PRIOR to the release.

So for now, you can remove these updates from the client & all the runtimes.